### DISCUSSION TITLE
MQTT Status for Pause and Resume Play

### DISCUSSION THREAD

**Discussion by BS-GIFHUB** [mikebrady/shairport-sync#1371] [General]
Hi, I installed the MQTT support and it works excellent so far. I'm looking for an information if the music is on or stops, because I like to use this nice software in a home automation envirement to switch on an amplifier and also off by status of the airplay stream. So if I stop the music I like to switch off the amplifier by the status of an MQTT message and after pressing play again I like to switch on the amplifier. 

I found the MQTT "prsm" which gives me the information that resume was clicked. But I didn't found any status/message that shows me that the music was stopped.  "pfls" doesn't work.

Any ideas or helps ?

Thanks Bernhard

**Comment by mikebrady** [comment]
Hi Bernhard. I think there is a feature of some MQTT clients that they suppress MQTT messages with no payload. You can now add a dummy payload in the configuration file -- build the `development` branch of Shairport Sync/
